src/SDKs/OperationalInsights/DataPlane/OperationalInsights.Data/OperationalInsights.Data.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft Azure Operational Insights Data Library</Description>
+    <PackageId>Microsoft.Azure.Management.OperationalInsights.Data</PackageId>
```

Comment:
@csuich2 if you are data plane, you should avoid using Management in your namespace, rather the package should be named as
Microsoft.Azure.OperationalInsights.
Data at the end of the namespace seems redundant.

Also name your csproj as OpertionalInsights.csproj, data in the project seems redundant. Because management plane project is already named as Management.OperationalInsights.csproj

Reply:
@shahabhijeet I can definitely remove the management part - that makes sense. I think removing the Data from the end would be bad, though. Shouldn't we make it clear this is the data plane package vs the management package? I suppose the management plane one already has 'management', but making it explicit seems like a good thing.

Reply:
@shahabhijeet Upon further investigation, I see that the convention appears to be management plane uses a management suffix and the data planes have no prefix or suffix. I'll make the change to the API spec README and regenerate the code.